### PR TITLE
Checksum validation added

### PR DESCRIPTION
**Pull request by francovenica**
When adding an address I add it with the toUpperCase() method, the value should be automatically checksummed so I compare the value in the input with the checksummed address.

Unrelated to this PR, I Added also in send_funds test a line that will check if the review button is disabled in a specific point during the filling of the send funds form. Is still commented because the fix has not been merged into develop. It should be uncommented once that fix is implemented into the gnosis safe app.

### CODE REVIEW

**Comment by francovenica** [comment]
This PR should NOT be merged until the #69 is merged and approved

File: src/add_remove_owner.test.js [hunk 1]
```diff
@@ -78,7 +78,10 @@ describe('Adding and removing owners', () => {
     await clearInput(settingsPage.add_owner_address_input, gnosisPage)
 
     console.log('Input valid owner name and address')
-    await clickAndType(settingsPage.add_owner_address_input, gnosisPage, newOwnerAddress)
+    // validating checksum
+    await clickAndType(settingsPage.add_owner_address_input, gnosisPage, newOwnerAddress.toUpperCase())
+    const NewOwnerAddressChecksummed = await getInnerText(settingsPage.add_owner_address_input, gnosisPage)
+    expect(NewOwnerAddressChecksummed).toEqual(newOwnerAddress)
```
[the remark this answers is not in the document]
Comment (francovenica) [reply src/add_remove_owner.test.js:84]: I always start them with lower case, idk why I did it that way this time :D . I'll change it.

File: src/create_safe.test.js [hunk 2]
```diff
@@ -65,7 +66,17 @@ describe('Create New Safe', () => {
...
+    await clickAndType(
+      { selector: createSafePage.address_field(1), type: 'css' },
+      gnosisPage,
+      owner2Address.toUpperCase(),
+    )
+    const OwnerAddressChecksummed = await getInnerText(
+      { selector: createSafePage.address_field(1), type: 'css' },
+      gnosisPage,
+    )
+    expect(OwnerAddressChecksummed).toEqual(owner2Address)
```
Comment (dasanra) [inline src/create_safe.test.js:79]: Didn't write before but here we have the same issue

Comment (francovenica) [reply src/create_safe.test.js]: Changed

**Pull request merged**